ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ntp-servers.j2

```diff
@@ -1,6 +1,8 @@
 {# eos - NTP Server #}
 {% if ntp_server is arista.avd.defined %}
+{% if ntp is not arista.avd.defined %}
```

Comment:
Is there any use case to have not defined for ntp? Afaik we only do defined for the functionality and only for the documentation we have not defined.

Reply:
Strangely molecule fails but, no errors...